R/getkey.R

```diff
@@ -35,32 +35,6 @@
 #' Note that NCBI Entrez doesn't require that you use an API key, 
 #' but you should get higher rate limit with a key, so do get one.
 #' 
-#' @section Tropicos:
-#' Get an API key at http://services.tropicos.org/help?requestkey
-#' 
-#' @section EOL:
-#' EOL requires an API key. You can pass in your EOL api
-#' key in the function call like
-#' \code{sci2comm('Helianthus annuus', key="<your eol api key>")}. You can
-#' also store your EOL API key in your .Rprofile file as
-#' \code{options(EOL_KEY = "<your eol api key>")}, or just for the current
-#' session by running \code{options(EOL_KEY = "<your eol api key>")} in
-#' the console.
-#'
-#' @section IUCN:
-#' IUCN requires an API key. See \code{\link[rredlist]{rredlist-package}} 
-#' for help on authentiating with IUCN Redlist, and 
-#' http://apiv3.iucnredlist.org/api/v3/token for requesting a key
-#' 
-#' @section NCBI Entrez:
-#' From NCBI's docs: "E-utils users are allowed 3 requests/second without an 
```

Comment:
@maelle i think we do want to retain this bit about 3 reqeusts per second without a key, and that using a key increases rate limit

Reply:
the other lines deleted are okay

Reply:
that part was added back in the docs of getkey